content/templates/_components/c-checkbox/01-c-checkbox.md

```diff
@@ -1,5 +1,3 @@
 ---
 title: Checkbox
```

Comment:
I'm noticing that most of the entries in the menu are plurals. Should this be change for consistency ?